app/helpers/kuroko2/job_definitions_helper.rb

```diff
@@ -34,5 +34,59 @@ def star_link_for(definition)
         )
       end
     end
+
+    def format_kuroko_script(script_text)
+      raw(
+        script_text.each_line.map { |line|
+          formatted_line = ERB::Util.html_escape(line)
+
+          if (matched = Kuroko2::Workflow::ScriptParser::LINE_REGEXP.match(line.chomp))
+            case matched[:type]
+            when 'wait'
+              formatted_line = format_wait_task(line, matched)
+            when 'sub_process'
+              formatted_line = format_sub_process_task(line, matched)
+            end
+          end
+
+          formatted_line
+        }.join('')
+      )
+    end
+
+    private
+
+    def format_wait_task(line, matched)
+      definition_names = []
+      formatted_line = line.gsub(Kuroko2::Workflow::Task::Wait::OPTION_REGEXP) { |option|
+        definition = Kuroko2::JobDefinition.find_by(id: $1.to_i)
+        if definition.present?
+          definition_names << definition.name
+          link_to(option, Kuroko2::Engine.routes.url_helpers.job_definition_path(definition.id))
```

Comment:
This is a helper method. Are you sure that `Kuroko2::Engine.routes.url_helpers` is required here?